client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+											data-dropdown-name="resources"
+											role="menu"
+											aria-label={ translate( ' Resources' ) }
+											aria-hidden="true"
+										>
+											<ul>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//en.support.wordpress.com/"
+														title={ translate( ' Support' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'WordPress.com Support' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//en.blog.wordpress.com/"
+														title={ translate( ' News' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'News' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/go/"
+														title={ translate( 'Website Building Tips' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Website Building Tips' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/business-name-generator/"
+														title={ translate( ' Business Name Generator' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Business Name Generator' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/logo-maker/"
+														title={ translate( ' Logo Maker' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Logo Maker' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/webinars/"
+														title={ translate( ' Daily Webinars' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'Daily Webinars' ) }
+													</a>
+												</li>
+												<li>
+													<a
+														role="menuitem"
+														className="x-dropdown-link x-link"
+														href="//wordpress.com/courses/"
+														title={ translate( 'WordPress Courses' ) }
+														tabIndex="-1"
+													>
+														{ translate( 'WordPress Courses' ) }
+													</a>
+												</li>
+											</ul>
+										</div>
+									</li>
+									<li className="x-nav-item x-nav-item--wide">
+										<a
+											role="menuitem"
+											className="x-nav-link x-link"
+											href="//wordpress.com/pricing/"
+											title={ translate( 'Plans' ) + ' & ' + translate( 'Pricing' ) }
```

Comment:
The entire "Plans & Pricing" string should be localized as one unit. Not all languages will use the "&" sign.